tests/template_tests/test_extends.py

```diff
@@ -117,3 +119,11 @@ def test_unique_history_per_loader(self):
         template = engine.get_template('base.html')
         output = template.render(Context({}))
         self.assertEqual(output.strip(), 'loader2 loader1')
+
+    @override_settings(INSTALLED_APPS=[
+        'template_tests.extended',
+        'template_tests.base',
+    ])
+    def test_get_extended_base_template_w_extended_included_template(self):
+        template = get_template("base_site.html")
```

Comment:
I think it would improve the readability of the test to use the approach of `test_unique_history_per_loader` and put the templates in the Python rather than having to look up those templates to see what's going on. Also, please use single quotes throughout, unless a string contains a double quote.